src/api/Models/ILayerCodeExtensions.cs

```diff
@@ -1,7 +1,14 @@
 ﻿namespace BDMS.Models;
 
+/// <summary>
+/// Provides extension methods for the ILayerCode interface.
+/// </summary>
 public static class ILayerCodeExtensions
```

Comment:
REQ: Umbenennen, da es kein Interface ist
```suggestion
public static class LayerCodeExtensions
```